### PR TITLE
Add BitcoinTrade

### PR DESCRIPTION
**Pull request by matheusazzi**

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -35,6 +35,12 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Bitcoin Trade
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion
    - name: BitcoinTrade
```

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -35,6 +35,12 @@ websites:
...
+      url: https://www.bitcointrade.com.br/
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion
      url: https://bitcointrade.com.br/
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Could you provide any documentation link that describes how to enable 2FA on BitcoinTrade? If not, could you provide any screenshots of the setup process alternate the login process? (with personal information blurred out)

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Travis output:
```
<------------ ERROR ------------>
1. _data/cryptocurrencies.yml is not alphabetized by name
```

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**